README.md

```diff
@@ -1,150 +1,52 @@
-<img src="src/assets/img/icon-128.png" width="64"/>
+<img src="./src/assets/img/icon-128.png" width="25">
 
-# Chrome Extension (MV3) Boilerplate with React 18 and Webpack 5
+# Finfriend - notes feature demo 
 
-[![npm](https://img.shields.io/npm/v/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
-[![npm-download](https://img.shields.io/npm/dw/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
-[![npm](https://img.shields.io/npm/dm/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
+Originally forked from : https://github.com/teziapp/clipboard-extension
 
-## Announcements
+**Working Features as of now :**
+- All previous features
```

Comment:
A Readme file should always be timeless. There is no previous/before in a readme file.